whylog/teacher/mock_outputs.py

```diff
@@ -1,37 +1,66 @@
-from whylog.teacher.user_intent import UserConstraintIntent, UserParserIntent, UserRuleIntent
+from whylog.teacher.user_intent import Group, UserConstraintIntent, UserParserIntent, UserRuleIntent
 
 
 def create_sample_rule():
-    # Constraint types
     identical_constr = "identical"
     different_constr = "different"
     hetero_constr = "hetero"
 
-    # convertions
     to_date = "date"
-    # to_float = "float"
-    """
-    content1 = "2015-12-03 12:08:09 Connection error occurred on alfa36. Host name: 2"
-    content2 = "2015-12-03 12:10:10 Data migration from alfa36 to alfa21 failed. Host name: 2"
-    content3 = "2015-12-03 12:11:00 Data is missing at alfa21. Loss = 567.02 GB. Host name: 101"
-    """
+    to_string = "string"
+    to_int = "int"
+
+    sample_line1 = "2016-04-12 23:54:45 Connection error occurred on comp1. Host name: host1"
+    sample_line2 = "2016-04-12 23:54:40 Data migration from comp1 to comp2 failed. Host name: host2"
+    sample_line3 = "2016-04-12 23:54:43 Data is missing at comp2. Loss = 150 GB. Host name: host2"
 
     regex1 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Connection error occurred on (.*)\. Host name: (.*)$"
     regex2 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Data migration from (.*) to (.*) failed\. Host name: (.*)$"
     regex3 = "^(\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d) Data is missing at (.*)\. Loss = (.*) GB\. Host name: (.*)$"
 
-    parser1 = UserParserIntent("hydra", regex1, [1], {1: to_date})
-    parser2 = UserParserIntent("hydra", regex2, [1], {1: to_date})
-    parser3 = UserParserIntent("filesystem", regex3, [1], {1: to_date})
+    groups1 = {
+        1: Group("2016-04-12 23:54:45", to_date),
+        2: Group("comp1", to_string),
+        3: Group("host1", to_string)
+    }
+    groups2 = {
+        1: Group("2016-04-12 23:54:40", to_date),
+        2: Group("comp2", to_string),
+        3: Group("host2", to_string)
+    }
+    groups3 = {
+        1: Group("2016-04-12 23:54:43", to_date),
+        2: Group("comp2", to_string),
+        3: Group("150", to_int),
+        4: Group("host2", to_string)
+    }
+
+    # resource location is temporary a string, because there is no ResourceLocation class yet.
+    # TODO: Change following resource locations to ResourceLocation objects
+    resource_location1 = "serwer1"
```

Comment:
Yes, but it is temporary, since ResourceLocation class doesn't exist. I will change it later as it is said in obove `TODO`